client/trino-client/src/test/java/io/trino/client/auth/external/TestExternalAuthenticator.java

```diff
@@ -238,13 +239,12 @@ public void testAuthenticationFromMultipleThreadsWithCachedTokenAfterAuthenticat
     @Test(timeOut = 2000)
     public void testAuthenticationFromMultipleThreadsWithCachedTokenAfterAuthenticateTimesOut()
     {
-        MockRedirectHandler redirectHandler = new MockRedirectHandler()
-                .sleepOnRedirect(Duration.ofMillis(5));
+        MockRedirectHandler redirectHandler = new MockRedirectHandler();
 
-        ExternalAuthenticator authenticator = new ExternalAuthenticator(redirectHandler, (uri, duration) -> TokenPollResult.pending(uri), KnownToken.memoryCached(), Duration.ofMillis(1));
```

Comment:
Does this refactor matter? If not I would revert.

Reply:
It looks like it belongs to the next commit.